### PR TITLE
New template for SCRs and one typo fix

### PR DESCRIPTION
**Pull request by pburkholder**
## Changes proposed in this pull request:
-
- Add a new issue template to help ensure we notify cors before starting an SC
-

## security considerations
Safe

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/conmon-3-scr.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+---
+name: Compliance - Start SCR
+title: "Initiate an SCR engagement"
+about: INTERNAL ONLY Start a new Significan Change engagement with 3PAO
```
Comment (Chiakao) [inline .github/ISSUE_TEMPLATE/conmon-3-scr.md]: Fix typo in "Significan"

**Review by Chiakao — APPROVED** [review]

**Comment by pburkholder** [comment]
@Chiakao I also found a typo. Can you re-approve?

**Review by jameshochadel — APPROVED** [review]
ty Peter!

**Comment by pburkholder** [comment]
I think I'll close this unmerged, as having yet another issue template may not really help here, and just clutters an already overlong list of issue types. I think we instead need to formalize our SCR pipeline, and have this as one of the steps.

**Pull request closed without merging**